debug.log

```diff
@@ -15,3 +15,4 @@
 [1130/214333.166:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
 [1201/201418.146:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
 [1201/204759.007:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
+[0104/212732.439:ERROR:directory_reader_win.cc(43)] FindFirstFile: Syst�m nem�e nal�zt uvedenou cestu. (0x3)
```

Comment:
I suggest to put this file to .gitignore :D

Reply:
It's weird.. I see it written .gitignore :D

Reply:
This is a little more complicated than just adding/removing from gitignore. Try to see if you can figure it out, it shouldn't be hard to google. Otherwise we'll catch up on this.